app/controllers/ReadProtectionsController.scala

```diff
@@ -119,4 +131,14 @@ class ReadProtectionsController @Inject() (
     )
   }
 
+  private def filterOnlyFixedProtection2016(model: TransformedReadResponseModel): Option[ProtectionModel] = {
+    val fixedProtectionType = model.activeProtection.filter(_.isFixedProtectionType2016)
+    if (fixedProtectionType.isEmpty) {
+      logger.error("There is no active Fixed Protection 2016")
```

Comment:
Last time I've heard, error-level log may raise some platform alerts. Is it something we want?